include/deal.II/optimization/line_minimization.h

```diff
@@ -242,9 +242,9 @@ namespace LineMinimization
    *
    *     // However, the corresponding gradient given in eq 5.14 is wrong. The
    *     // suggested result
-   *     // <code>const double g = -(residual_0*residual_trial);</code>
+   *     // const double g = -(residual_0*residual_trial);
    *     // should actually be
-   *     // $g = G(V + alpha*delta)*[ K(V + alpha*delta)*delta$.
+   *     // g = G(V + alpha*delta)*[ K(V + alpha*delta)*delta.
```

Comment:
doxygen was choking on code and formulas inside a code block here. This ate the documentation of the rest of the file. Ooops.